packages/charts/README.md

```
@@ -0,0 +1,21 @@
+# Breeze Charts
+
+_React chart components, built with Tailwind CSS and VisX_
+
+[![codecov](https://codecov.io/gh/jszymanowski/breeze/branch/main/graph/badge.svg)](https://codecov.io/gh/jszymanowski/breeze)
+
+---
+
+### Releases & Publishing
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)